diff --git a/drivers/hwmon/nct6775.c b/drivers/hwmon/nct6775.c
index 7f4dce063ff3..f90011327fa2 100644
--- a/drivers/hwmon/nct6775.c
+++ b/drivers/hwmon/nct6775.c
@@ -4994,6 +4994,7 @@ static const char * const asus_wmi_boards[] = {
 	"ROG STRIX B550-F GAMING",
 	"ROG STRIX B550-F GAMING (WI-FI)",
 	"ROG STRIX Z490-I GAMING",
+	"ROG STRIX X570-E GAMING",
 	"TUF GAMING B550M-PLUS",
 	"TUF GAMING B550M-PLUS (WI-FI)",
 	"TUF GAMING B550-PLUS",
